### PR TITLE
Extend types with receivedAt

### PR DESCRIPTION
**Pull request by francamps**

### CODE REVIEW

**Review by gregorybchris — APPROVED** [review]

File: packages/react/src/index.ts [hunk 1]
```diff
@@ -29,16 +29,30 @@ export type {
...
+  ChatMetadata as OriginalChatMetadata,
+  SubscribeEvent as OriginalSubscribeEvent,
+  ToolCallMessage as OriginalToolCallMessage,
+  ToolErrorMessage as OriginalToolErrorMessage,
+  ToolResponseMessage as OriginalToolResponseMessage,
+  UserInterruption as OriginalUserInterruption,
+  UserMessage as OriginalUserMessage,
 } from 'hume/api/resources/empathicVoice';
+
+type WithReceivedAt<T> = T & { receivedAt: Date };
```
Comment (yinishi) [inline packages/react/src/index.ts:46]: delete lines 15-28

rename types on 48-58 with the types that are exported in 15-28

Comment (francamps) [reply packages/react/src/index.ts:31]: done

File: packages/react/src/index.ts [hunk 2]
```diff
@@ -42,3 +27,17 @@ export type {
...
+export type AssistantTranscriptMessage = WithReceivedAt<AssistantMessage>;
+export type AudioMessage = WithReceivedAt<AudioInput>;
+export type AudioOutputMessage = WithReceivedAt<AudioOutput>;
+export type ChatMetadataMessage = WithReceivedAt<ChatMetadata>;
+export type JSONMessage = WithReceivedAt<SubscribeEvent>;
+export type ToolCall = WithReceivedAt<ToolCallMessage>;
+export type ToolError = WithReceivedAt<ToolErrorMessage>;
+export type ToolResponse = WithReceivedAt<ToolResponseMessage>;
+export type UserInterruptionMessage = WithReceivedAt<UserInterruption>;
+export type UserTranscriptMessage = WithReceivedAt<UserMessage>;
```
Comment (yinishi) [inline packages/react/src/index.ts:43]: JSONErrorMessage is missing

Comment (francamps) [reply packages/react/src/index.ts:45]: readded

**Review by yinishi — APPROVED** [review]

**Pull request merged**